pkgs/applications/science/math/palp/darwin.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/GNUmakefile b/GNUmakefile
+index 4cac75f..5319fe5 100644
+--- a/GNUmakefile
++++ b/GNUmakefile
+@@ -15,7 +15,7 @@ MORI_SRC= MoriCone.c SingularInput.c
+ MORI_OBJ= $(MORI_SRC:.c=.o)
+ 
+ 
+-CC=gcc
++CC=cc
```

Comment:
I'd use `substituteInPlace` instead of a patch, it's pretty safe in this case and more likely to continue working in the future.